### ISSUE TITLE
OpenSubtitle Issues

### ISSUE THREAD

**Issue by komizha** [iina/iina#2586]
**System and IINA version:**

- macOS 
- IINA 

**Expected behavior:**

Searching subtitles through OpenSubtitles.org

**Actual behavior:**

Can not search Anonymously, and can not login into opensubtitles.org through iina. If I try to search for subtitles I am getting Network Error and if I am trying to login into opensubtitles.org through the iina I am getting "Check Username/password" error. Anyone knows how to solve this? Thanks

**How often does this happen?**

All the time.

**Comment by komizha** [comment]
I succeeded to login into opensubtitles account but when I click find online subtitles nothing happens. I don't get any error or anything it just stays the same as button has not been pressed.

**Comment by megadr01d** [comment]
Yep, nothing happens for me too, without a OpenSubtitles account. No success, no error, no nothing when searching for subtitles.

**Comment by firewater2** [comment]
Same

**Comment by Shiyazudeen** [comment]
Mac 10.15.6 - IINA 1.0.7 - same problem. Even with OpenSubtitles account, the srt files get listed, but on download, always get the 'Network Error'. Any fixes or updates?

**Comment by engineerbaraazakariya** [comment]
same error 

macos 10.15.6 
1.1.0-beta1 Build 121
it doesnot open any window if i press cmd+shift+D then nothing got shown just network error message and that's it instead of listing the subtitle

**Comment by frddr** [comment]
Same problem with OpenSubtitles.org
When stopping watching video I close the lid (usually I exit iina pressing Q). When next day I open the MacBook again and want to watch, 30% of the time the sub search doesn't work. Pressing cmd+shift+D either doesn't do anything or displays network error. To solve it I need to logout and back in. In 60% of the cases this works. In 40% of the cases I still had no effect and I need to reboot the MacBook to get it working again.

**Comment by wb127** [comment]
mac os 12.6.1
1.3.1 Build 133
 I can success login opensubtitle's account but when I choose search UI show subtitle from opensubtitle and then back with read file error 读取文件错误

**Comment by HoreaGeorgian** [comment]
This shit stil not resolved.

**Comment by Rohindh-R** [comment]
This still happens all the time

**Comment by tangxuan** [comment]
same, I'm sure of valid account.
`Error, invalid username/password  `

**Comment by xiaotianxt** [comment]
Same here. By the way, there are opensubtitles.com and opensubtitles.org; I've tried both, and neither works.

**Comment by balcis** [comment]
opensubtitles asks captcha at login so it can not login with either username or mail address.
